code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -311,7 +343,14 @@ There are several things that need to be remembered:
 		if(!t_state)
 			t_state = belt.icon_state
 
-		var/image/standing = belt.build_worn_icon(state = t_state, default_layer = BELT_LAYER, default_icon_file = 'icons/mob/belt.dmi')
+		var/image/standing
+		if(belt.icon_override)

```

Comment:
stuff like this should all go inside of ```build_worn_icon()```
Remember to give highest priority to ```alternate_worn_icon```, as it's the admin/highest override.